### ISSUE TITLE
Can't save when I change "save path options"...

### ISSUE THREAD

**Issue by zoutroy** [jscher2000/Save-webP-as-extension#18]
Hi ! 
All is in the title. When I change this option for "directly in my downloads folder", I can't download images anymore. Like in the previous message, only IE11, GIF(V), Options and the closing cross functions. All image formats are inactive. When I change it back to "Save_webP_folder", it works again, but there are a lot of folders to click on to reach the image, which is pretty boring.
I must precise that It worked perfectly well until yesterday, but I had to reset Firefox, and since then I had this problem. Great tool, by the way, very useful... 
(Excuse my english, I'm french)

**Comment by jscher2000** [comment]
Hi, thank you for this report. I'm trying to think of a way to see whether Firefox is generating any error messages. But I'll come back to that in a future reply if needed.

I personally don't notice a problem if the only change I make is between "Save_webP" and "Directly in my Downloads". In case it matters, I'm using the standard Windows Downloads folder and not a different folder.

Are you using subfolders or any of the fancier naming options? That will help me test different combinations.

Regarding finding images, I would start from the Downloads list (recent ones on the toolbar button drop-down, older ones under Ctrl+J (on Mac, it's probably Command+J)), where the folder icon on the right side of the listing will pop open the file location in Windows File Explorer (or Mac Finder).

**Comment by zoutroy** [comment]
No, I use the main download folder, and the icon on the right side leads me to the usual "downloads" folder.

**Comment by jscher2000** [comment]
In version 1.3.3, I added some more error handling to hopefully show why a save failed Look for a popup message on the page where you clicked the button to save the image. Could you let me know what you see there, if anything?
